src/main/resources/assets/titanium/lang/en_us.json

```diff
@@ -26,5 +26,9 @@
   "direction.titanium.south": "South",
   "direction.titanium.up": "Up",
   "direction.titanium.down": "Down",
-  "itemGroup.resources": "Titanium: Resources"
-}
\ No newline at end of file
+  "itemGroup.resources": "Titanium: Resources",
+  "tooltip.titanium.redstone.ignored": "Redstone Mode: Ignored",
+  "tooltip.titanium.redstone.no_redstone": "Redstone Mode: Run with no redstone signal",
+  "tooltip.titanium.redstone.redstone": "Redstone Mode: Run with redstone signal",
+  "tooltip.titanium.redstone.once": "Redstone Mode: Run Once"
```

Comment:
Redstone Mode: Run Once -> Redstone Mode: Pulse

Reply:
Run once on pulse?

Reply:
"Run on Pulse" in that case.